kythe/web/site/getting-started-macos.md

```diff
@@ -63,19 +63,6 @@ for pkg in asciidoc bison brotli flex go graphviz leveldb node openjdk parallel
    brew install $pkg
 done
 
-# For Bazel, we recommend using Bazelisk, a wrapper tool that uses
```

Comment:
This leaves macOS users without clear instructions for how to install Bazel. What's the current recommendation, if not to use Bazelisk?

Reply:
Oh, good catch.

Reply:
Wait ... that is on the main "getting started". The "getting started for macOS" is about dependencies. Bazel install instructions are general ("surf to bazel.io and install")

Reply:
Fair enough. That didn't used to work. Interestingly, one of their four recommended ways to install Bazel on macOS is via Bazelisk.

(I don't object to omitting the instructions, but I did have to drill about four pages deep into the mess of bazel.io to find that)